setup.cfg

```diff
@@ -84,7 +84,6 @@ exclude_lines =
   def autofill_value
   def is_autofill_possible
   def require_setting
-  if ext in exts
```

Comment:
It would have been better if you would have run moban first without rebasing so this would have gone in the first commit and then the rebasing of the moban repo and running moban again to make the moban sync commit. But nevertheless.